src/quacc/recipes/mlp/_base.py

```diff
@@ -56,10 +62,20 @@ def pick_calculator(
 
         calc = CHGNetCalculator(**kwargs)
 
+    elif method.lower() == "mace-mp-0":
+        from mace import __version__
+        from mace.calculators import mace_mp
+
+        if "default_dtype" not in kwargs:
+            kwargs["default_dtype"] = "float64"
+        calc = mace_mp(**kwargs)
+
     elif method.lower() == "mace":
         from mace import __version__
         from mace.calculators import mace_mp
 
+        mace_warning()
```

Comment:
Ah, while I originally said we should use `monty.dev.requires`, that function (in hindsight) really is best suited as a decorator on the parent recipe. Since that doesn't make much sense here, I suggest just logging a warning with the `logging` utility instead of calling a custom `mace_warning()` function.